homeassistant/components/enphase_envoy/manifest.json

```
@@ -2,13 +2,17 @@
   "domain": "enphase_envoy",
   "name": "Enphase Envoy",
   "documentation": "https://www.home-assistant.io/integrations/enphase_envoy",
-  "requirements": ["envoy_reader==0.18.4"],
-  "codeowners": ["@gtdiehl"],
+  "requirements": [
+    "envoy_reader==0.19.0"
+  ],
+  "codeowners": [
+    "@gtdiehl"
+  ],
   "config_flow": true,
   "zeroconf": [
     {
       "type": "_enphase-envoy._tcp.local."
     }
   ],
   "iot_class": "local_polling"
-}
+}
```

Is this on purpose ? as far as I can see it do no harm, but GitHub flags it.
```suggestion
}

```

---

That doesn't matter, it depends a bit which formatter was used for the JSON file. We don't need to block review/merge on this.